jdk17/jaxrs-panache/pom.xml

```diff
@@ -35,5 +35,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jdbc-postgresql</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why is explicit use of the DB needed and why don't DevServices work OOB in this case?

Reply:
Is not really "necessary". In other TS as Quarkus-test-suite, devMode scenarios are not running in native mode. I think that makes sense this decision, I mean... DevMode is more for...development, where you are changing runtime properties, devservices...etc and all of this stuff, doesn't really "work" with a native application where you need to compile the app in order to take into account these changes.